doc/system_requirements.md

```diff
@@ -108,13 +146,13 @@ Covers:
 
 Needs: dsn
 
-### Update extension
+#### Update extension
 `req~update-extension~1`
 
 When updating an extension that is already installed in an older version, EM checks if any parameter definition has changed. If there were breaking changes, EM cannot perform the update automatically and aborts the installation with an appropriate error message.
 
 (( how to define "breaking" changes?
-any change in paramters is a breaking change
+any change in parameters is a breaking change
```

Comment:
That typo happens to me also on a regular basis. ;-)

Reply:
😆